pages.es/common/typeinc.md

```diff
@@ -0,0 +1,21 @@
+# typeinc
+
+> Un programa de prueba de velocidad de tecleo en terminal basado en `ncurses`, escrito en python.
+> Prueba diferentes niveles de dificultad y mejora tu velocidad de tecleo.
+> Más información: <https://github.com/AnirudhG07/Typeinc>.
+
+- Entra en el test de mecanografía:
+
+`typeinc`
+
+- Muestra la lista de los 10 mejores para el nivel de dificultad de entrada:
+
+`typeinc --ranklist {{nivel_de_dificultad}}`
+
+- Obtiene palabras aleatorias en inglés presentes en nuestra lista de palabras:
+
+`typeinc --words {{conteo_de_palabras}}`
+
+- Calcula el re[s]ultado hipotético en Typeinc:
+
+`typeinc --score`
```

Comment:
```suggestion
`typeinc -s`
```

Shouldn't the option be short to match the mnemonic?

https://github.com/AnirudhG07/Typeinc/blob/main/typeinc/main.py#L69

Reply:
> Shouldn't the option be short to match the mnemonic?
> 
> [AnirudhG07/Typeinc@`main`/typeinc/main.py#L69](https://github.com/AnirudhG07/Typeinc/blob/main/typeinc/main.py?rgh-link-date=2024-06-06T19%3A29%3A45Z#L69)

I don't think so, we have examples like this on many pages.

Reply:
I do not understand, could you please elaborate?

Reply:
> I do not understand, could you please elaborate?

Previously, we had pages using mnemonics to highlight the long option of short option or the short option's meaning so due to the method's inconsistencies, we discussed and introduced this new syntax `{{-s|--score}}` to showcase both short and long options beside each other (which is much clearer in most cases).